TTTSwiftCommandLine/ConsoleMessages.swift

```diff
@@ -3,19 +3,19 @@ public class ConsoleMessages {
     public init() {}
     
     public func welcomeMessage() -> String {
-        return "\nWelcome to Tic-Tac-Toe! Would You Like to Start a New Game?\n1) Yes\n2) No"
+        return "\nWelcome to Tic-Tac-Toe! What would you like to do?\n1) Play Human vs. Human\n2) Play Human vs. Computer\n3) Play Computer vs. Human\n4) Exit Tic-TacToe"
```

Comment:
Does Swift support multiline strings? I don't think it does, but if so, it might be nice to break this into multiple lines.